Configuration/Geometry/python/GeometrySimDB_cff.py

```diff
@@ -4,12 +4,14 @@
 # Geometry master configuration
 #
 # Ideal geometry, needed for simulation
-from GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff import *
-from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
-from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
-from Geometry.EcalCommonData.ecalSimulationParameters_cff   import *
-from Geometry.HcalCommonData.hcalSimulationParameters_cfi   import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
-from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
-from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
-from Geometry.MuonNumbering.muonGeometryConstants_cff       import *
+
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+
+def _loadSimGeomDBDDD(process) :
+    process.load('Configuration.Geometry.GeometryDDDSimDB_cff')
+
+def _loadSimGeomDBDD4hep(process) :
+    process.load('Configuration.Geometry.GeometryDD4hepSimDB_cff')
+
+modifyGeometryConfiguration = (~dd4hep).makeProcessModifier(_loadSimGeomDBDDD)
+modifyGeometryConfiguration_dd4hep = dd4hep.makeProcessModifier(_loadSimGeomDBDD4hep)
```

Comment:
I'm wondering if organization along
```py
# common
from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
from Geometry.EcalCommonData.ecalSimulationParameters_cff   import *
from Geometry.MuonNumbering.muonGeometryConstants_cff       import *

def _loadSimGeomDBDDD(process):
    process.load("GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff")
    process.load("Geometry.MuonNumbering.muonNumberingInitialization_cfi")
    process.load("Geometry.HcalCommonData.hcalSimulationParameters_cfi")
    process.load("Geometry.HcalCommonData.hcalDDDSimConstants_cfi")
    process.load("Geometry.HcalCommonData.hcalSimulationConstants_cfi")
    process.load("Geometry.HcalCommonData.caloSimulationParameters_cff")

def _loadSimGeomDBDD4hep(process):
    process.DDSpecParRegistryESProducer = cms.ESProducer("DDSpecParRegistryESProducer")
    process.DDVectorRegistryESProducer = cms.ESProducer("DDVectorRegistryESProducer")
    process.DDCompactViewESProducer = cms.ESProducer("DDCompactViewESProducer")

    process.load("DetectorDescription.DDCMS.DDDetectorESProducerFromDB_cfi")
    process.load("Geometry.HcalCommonData.hcalSimDBConstants_cff")

modifyGeometryConfiguration = (~dd4hep).makeProcessModifier(_loadSimGeomDBDDD)
modifyGeometryConfiguration_dd4hep = dd4hep.makeProcessModifier(_loadSimGeomDBDD4hep)
```
would be more clear or less clear.

Reply:
`GeometryDDDSimDB.py` is exactly identical to the original `GeometrySimDB_cff.py`. That means for all existing non-DD4hep simulation workflows there is strong assurance that there will be no change in behavior. Since these configs are used so widely in production, I would like to minimize changes to reduce opportunities for errors.